src/uwtools/scheduler.py

```diff
@@ -222,20 +222,23 @@ def pre_process(self) -> Dict[str, Any]:
     def select(self, items) -> Dict[str, Any]:
         """select logic"""
         total_nodes = items.get(RequiredAttribs.NODES, "")
-        cores_per_node = items.get(RequiredAttribs.TASKS_PER_NODE, "")
-        threads_per_core = items.get(OptionalAttribs.THREADS, "")
+        tasks_per_node = items.get(RequiredAttribs.TASKS_PER_NODE, "")
+        # Set default threads=1 to address job variability with PBS
+        threads = items.get(OptionalAttribs.THREADS, 1)
         memory = items.get(OptionalAttribs.MEMORY, "")
 
-        items["-l select"] = f"={total_nodes}:mpiprocs={cores_per_node}"
-        if threads_per_core not in NONEISH:
-            items[
-                "-l select"
-            ] = f"{items['-l select']}:ompthreads={threads_per_core}:ncpus={int(cores_per_node) * int(threads_per_core)}"
+        select = [f"{total_nodes}",
```

Comment:
This variable has the same name as the function. Please choose a different variable name so that the outer scope isn't redefined.

Reply:
Yes. That would be my oversight.